langchain4j-oracle/README.md

```diff
@@ -13,6 +13,69 @@ This module implements `EmbeddingStore` using Oracle Database.
 </dependency>
 ```
 
+## Usage
```

Comment:
Shouldn't this info be in `docs/docs/integrations/embedding-stores/oracle.md` instead?

Reply:
I have added it to both files